Controller/EmployeeController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Controller;
+
+use Interface\EmployeeInterface;
+
+class Employee
```

Comment:
sử dụng implement class khi muốn triển khai sự phụ thuộc trong OOP
class Employee implement  EmployeeInterface

Reply:
Theo em hiểu Model Employee đã implement EmployeeInterface rồi,
ở class Employee trong EmployeeController sử dụng:
public function __construct(EmployeeInterface $employeeInterface)
    {
        $this->employeeModel = $employeeInterface;
    }  
thì vẫn cần implement EmployeeInterface nữa ạ ?

Reply:
thế xem lại phần tính chất của OOP cho anh nhé

Reply:
Em vẫn chưa hiểu ạ